src/libraries/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -833,6 +833,15 @@ public async Task Kill_ExitedNonChildProcess_DoesNotThrow(bool killTree)
                         // process still exists, wait some time.
                         await Task.Delay(100);
                     }
+
+                    DateTime now = DateTime.UtcNow;
+                    if (start.Millisecond + Helpers.PassingTestTimeoutMilliseconds <= now.Millisecond)
```

Comment:
This isn't right -- `Millisecond` wraps every second 🙂 I think you need to subtract one DateTime from the other

Reply:
Or easier than bothering with DateTime and TimeSpan, just use `Stopwatch` - start it at the top, then read off `ElapsedMilliseconds` whenever you need it. Don't need to bother stopping it.

Reply:
ElapsedMilliseconds may be what I was thinking of and used before. I switched to Ticks.